<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDialogPKCS12</class>
 <widget class="QDialog" name="CDialogPKCS12">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelFriendlyName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Friendly name :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFriendlyName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelCertificate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Certificate CN :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCertificate">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password :</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetCA">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>341</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>111</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonLoadCert">
      <property name="text">
       <string>Load cert...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSelectFromStack">
      <property name="text">
       <string>Select from stack</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonImportCert">
      <property name="text">
       <string>Import selected</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonPushAll">
      <property name="text">
       <string>Push all in stack</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSaveAs">
      <property name="text">
       <string>Save As...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonImportMain">
      <property name="text">
       <string>Import cert &amp;&amp; key</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
